engines/router/Dockerfile

```diff
@@ -35,6 +35,7 @@ FROM alpine:latest
 ARG BIN_NAME=turing-router
 ENV BIN_NAME ${BIN_NAME}
 
+RUN apk update && apk add tzdata
```

Comment:
A comment above this line will be useful - to explain that this dependency is not required by the router, but it's a basic dependency for the experiment engine.